css/style.css

```diff
@@ -179,6 +179,23 @@ span.sortable-ghost {
   vertical-align: middle;
 }
 
+#controls button{
+  margin: 0.4em 0;
+  padding: 0.5em;
+  color: var(--color-light);
+  background-color: var(--color-button);
+  border: 0.2em solid var(--color-light);
+  outline: 0.2em solid var(--color-button);
```

Comment:
I think the `border`+`outline` combination is not needed here. For the color buttons, the border serves the purpose of displaying the currently selected color.